docs/demography.md

```diff
@@ -1882,14 +1882,14 @@ demography = msprime.Demography._ooa_model()
 debug = demography.debug()
 
 T = np.concatenate([
-        np.linspace(0, 1000, 2001),
-        np.linspace(1000, 1e4, 401)[1:]
+    np.linspace(0, 1000, 2001),
+    np.linspace(1000, 1e4, 401)[1:]
 ])
 
 R = np.zeros((len(T), 3))
-R[:,0], _ = debug.coalescence_rate_trajectory(T, [2, 0, 0, 0, 0, 0])
-R[:,1], _ = debug.coalescence_rate_trajectory(T, [0, 2, 0, 0, 0, 0])
-R[:,2], _ = debug.coalescence_rate_trajectory(T, [0, 0, 2, 0, 0, 0])
+R[:,0], _ = debug.coalescence_rate_trajectory(T, {"YRI": 1})
```

Comment:
These should have `2` instead of `1` because a single lineage can't coalesce with itself. To help out the user we could emit an error if the number of samples is not at least two?

Reply:
These are diploids though